src/server.ts

```
@@ -498,7 +561,11 @@ class CodeRabbitMCPServer {
             
           default:
             // Delegate to individual tools
+            logger.debug(`Delegating tool ${name} to orchestrator`);
             const toolResult = await this.orchestrator.executeTool(name, args);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)